model-engine/tests/unit/domain/test_llm_use_cases.py

```diff
@@ -483,6 +486,40 @@ async def test_completion_sync_use_case_predict_failed(
     assert response_1.output is None
 
 
+@pytest.mark.asyncio
+async def test_completion_sync_use_case_predict_failed_with_errors(
+    test_api_key: str,
+    fake_model_endpoint_service,
+    fake_llm_model_endpoint_service,
+    llm_model_endpoint_sync_tgi: Tuple[ModelEndpoint, Any],
+    completion_sync_request: CompletionSyncV1Request,
+):
+    fake_llm_model_endpoint_service.add_model_endpoint(llm_model_endpoint_sync_tgi[0])
+    fake_model_endpoint_service.sync_model_endpoint_inference_gateway.response = SyncEndpointPredictV1Response(
+        status=TaskStatus.SUCCESS,
```

Comment:
Probably outside of the scope of this PR but I was hoping we could get rid of `status` in the response, since we handle errors separately anyway.